website/docs/integrator-section/verifiable-document/did/revoking-document.md

```diff
@@ -14,7 +14,7 @@ It is possible to revoke a signed document only if you fulfill certain **prerequ
 - Some kind of DID documents:
   1. Base DID document (`ISSUER_IDENTITY`: `identityProof.identifier` against `ethr` DID)
   2. DID-DNS document (`ISSUER_IDENTITY`: `identityProof.identifier` against a [DNS-TXT](/docs/docs-section/how-does-it-work/issuance-identity))
-- A deployed `documentStore`, click [here](/docs/integrator-section/verifiable-document/ethereum/document-store) for how to do so.
+- A deployed `documentStore`, click [here](/docs/integrator-section/verifiable-document/ethereum/document-store) for the steps.
```

Comment:
this is linking to the `ethereum` section. Since we have a DID section for deploying the document store should we link to that instead?

Reply:
Updated to the DID tutorial